Gemfile.lock

```diff
@@ -242,96 +242,95 @@ GEM
       rest-client (>= 1.6.7)
     marcel (0.3.3)
       mimemagic (~> 0.3.2)
-    method_source (0.9.2)
-    mime-types (3.3)
+    method_source (1.0.0)
+    mime-types (3.3.1)
       mime-types-data (~> 3.2015)
     mime-types-data (3.2019.1009)
-    mimemagic (0.3.3)
-    mini_magick (4.9.5)
+    mimemagic (0.3.4)
+    mini_magick (4.10.1)
     mini_mime (1.0.2)
     mini_portile2 (2.4.0)
-    minitest (5.13.0)
-    msgpack (1.3.1)
+    minitest (5.14.0)
+    msgpack (1.3.3)
```

Comment:
### [msgpack](http://msgpack.org/)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.3.1 → 1.3.3

[_~~change-log~~, [source-code](https://github.com/msgpack/msgpack-ruby)_]





<details>
<summary>Commits</summary>

A change of **9** commits. See the full changes on [the compare page](https://github.com/msgpack/msgpack-ruby/compare/v1.3.1...v1.3.3).

These are the individual commits:
- (0b1bf90) [Fix unnecessary T_HASH check](https://github.com/msgpack/msgpack-ruby/commit/0b1bf902375283985fb0963143e5f9528b2767df)
- (94de501) [Merge pull request #174 from martelogan/fix-unnecessary-t_ha…](https://github.com/msgpack/msgpack-ruby/commit/94de501e28a6c892f73030323bbbe34dd377753b)
- (16e73bd) [#177 Added support for Ruby-2.7](https://github.com/msgpack/msgpack-ruby/commit/16e73bdba9cae1b4170ab63f98276492dd1bea28)
- (7107a14) [Merge pull request #178 from francesco-loreti/#177](https://github.com/msgpack/msgpack-ruby/commit/7107a14ced33b5b1f8c4aca13017ab340c0a6863)
- (4585633) [v1.3.2](https://github.com/msgpack/msgpack-ruby/commit/45856333a0ec2f9199011ec714fdbffe4a83ce20)
- (1052a8e) [make sure to check the rake-compiler-dock version](https://github.com/msgpack/msgpack-ruby/commit/1052a8eca4903105a2e33ed1a25faebe7bb85fc8)
- (3994aea) [update rake-compiler-dock for Ruby 2.7](https://github.com/msgpack/msgpack-ruby/commit/3994aea20291c4061047dd6b98e1aa38a4e33f42)
- (14af53d) [v1.3.3](https://github.com/msgpack/msgpack-ruby/commit/14af53dff7e2a26e3f2c42f9edfb2bdf8bd34e80)
- (64db1ba) [Merge pull request #179 from msgpack/update-rake-compiler-do…](https://github.com/msgpack/msgpack-ruby/commit/64db1bac56359c9257d22f3ffb7506b578d76741)


</details>